shared/bash_remediation_functions/replace_or_append.sh

```diff
@@ -71,7 +71,7 @@ function replace_or_append {
 
   # If the key exists, change it. Otherwise, add it to the config_file.
   if `grep -qi $key $config_file` ; then
-    $sed_command "s/$key.*/$formatted_output/g" $config_file
+    eval $sed_command "s/$key.*/$formatted_output/g" $config_file
```

Comment:
Good question.

I have updated the remediation for RHEL7 in #2110 and tested adding space to `key` and `format`, and it works, :)

`/etc/default/useradd` was added the following line
```
# Per CCE-CCE-27355-7: Set IN ACTIVE = 90 in /etc/default/useradd
IN ACTIVE = 90
```

And just found out that we add string `CCE-` twice... but that goes to another PR.